.github/workflows/build.yml

```diff
@@ -70,3 +82,7 @@ jobs:
           push: ${{ github.event_name != 'pull_request' && github.repository == 'photoview/photoview' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
+          build-args: |
+            BUILD_DATE=${{ steps.args.outputs.BUILD_DATE }}
```

Comment:
oh, I see(
I found the `${{ github.event.head_commit.timestamp }}`, but (according to the tooltip) it returns a Unix timestamp instead of a human-readable date-time with no built-in way to convert it without an additional step(
If that is true, then we still need this additional `run` step

Reply:
I tried with `github.event.repository.updated_at`, but it's not the same as the time running the job.

![image](https://github.com/user-attachments/assets/171963e1-0807-45a8-9470-e8e5ea1eae7b)

In my opnion, we should use `date` to get the right time here.

Reply:
Done.

Reply:
I don't know it from the beginning. I agreed that and I updated the build script to generate `BUILD_DATE`.

Reply:
Done.

Reply:
Great, thanks)